arbeitszeit_web/api/controllers/login_company_api_controller.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+from typing import List
+
+from arbeitszeit.use_cases.log_in_company import LogInCompanyUseCase
+from arbeitszeit_web.api.controllers.expected_input import ExpectedInput, InputLocation
+from arbeitszeit_web.api.response_errors import BadRequest
+from arbeitszeit_web.request import Request
+
+
+@dataclass
+class LoginCompanyApiController:
+    @classmethod
+    def create_expected_inputs(cls) -> List[ExpectedInput]:
+        return [
+            ExpectedInput(
+                name="email",
+                type=str,
+                description="Email.",
+                default=None,
+                location=InputLocation.form,
+                required=True,
+            ),
+            ExpectedInput(
+                name="password",
+                type=str,
+                description="Password.",
+                default=None,
+                location=InputLocation.form,
+                required=True,
+            ),
+        ]
+
+    request: Request
+
+    def create_request(self) -> LogInCompanyUseCase.Request:
+        email = self.request.get_form("email")
+        password = self.request.get_form("password")
+        if not email or not password:
+            raise BadRequest(message="Email or password missing.")
```

Comment:
Maybe for the future — it might be nice to distinguish those two error conditions somehow.  Also, if it's just a matter of empty fields, can we not have a client-side (CSS or JS) check?